src/components/CheckBox/HiddenCheckBox.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+import CheckMark from './CheckMark';
+
+const HiddenCheckBox = styled.input.attrs({
```

Comment:
Would name it **Input** and pass type directly in Checkbox file: `<Input type="checkbox" checked={checked} onChange={onChange} />`.
If we are creating controlled component in react passing *onChange* handler will be required. Both *checked* and *onChage* will come from parent form most likely.